dashboard_test.go

```diff
@@ -51,7 +51,8 @@ func TestView_MarshalJSON(t *testing.T) {
     "xColumn": "",
     "yColumn": "",
     "shadeBelow": false,
-    "position": ""
+		"position": "",
```

Comment:
SPACES SHALL RULE THE WEB

Reply:
it's weird that this is the way it appears on github since it was fine on VSCode